### ISSUE TITLE
Grass growth is not working

### ISSUE THREAD

**Issue by davidchan13** [Skyrat-SS13/Skyrat13#2041]
## Round ID: N/A

 (Numerous rounds spanning the last several months)

## Date and Time of Issue:

 N/A (Numerous rounds spanning the last several months)

## What Map did this Issue occur on: 

All (Box, Meta, Puby, Delta)

## Testmerges (If Applicable):

N/A no consistent test merge has been present for this bug.

## Reproduction:

 Grass growth is entirely RNG and unreliable. 

Grass has a maturation of 2, and a default production of 5. This is supposed to mean that after 7 cycles, it will be ready for it's first harvest.

This is not the case. Vanilla grass, gene edited and chemical enhanced grass alike, there is no rhyme or reason to when grass ready to harvest. Typical wait can be 50+ growth cycles with cases of age 100 grass still not having a single harvest. 

This also applies to mutate strains of grass (Carpet and Fairygrass)

This is troublesome as grass has it's uses to botany and public garden users who might want to make grass floor tiles and/or harvest grass to produce sand to create sandstone and dirt piles to grow more plants with.

TL;DR:
Plant grass seeds.
Use data analyzer to check stats (2 maturation + 5 production = harvest after 7 cycles)
Wait 10 times that number while tending plant and watching it meticulously.
Still not recieve any harvest.

**Comment by davidchan13** [comment]
I've attempted to back track the problem with little luck. I believe it's directly tied to the maturation score of 2, where as all other plants have 3 or more. Someone mentioned something about growth phases for plants and grass having trouble with them because of how growth works. 

TL;DR I believe raising the maturation stat to 3 would solve the issue.

**Comment by John-Peppercorn** [comment]
I figured out the problem. Something in grasses code makes it fucky with Robust Harvest. It will never reach full growth after being planted in a tray where robust harvest was used last.
